.lando.yml

```diff
@@ -198,8 +198,6 @@ services:
 proxy:
   mailhog:
     - mail.lndo.site
-  elasticsearch:
-    - elasticsearch.lndo.site:9200
```

Comment:
this failed locally with the latest version of lando. Can't think of a good reason to have this since localhost:9200 does the same. I removed it.